platform-docs/docs/create-block/block-supports.md

```diff
@@ -1,3 +1,32 @@
 ---
-sidebar_position: 4
+sidebar_position: 3
 ---
+
+# Block Supports
+
+A lot of blocks, including core blocks, offer similar customization options. Whether that is to change the background color, text color, or to add padding, margin customization options…
+To avoid duplicating the same logic over and over in your blocks and to align the behavior of your block with core blocks, Gutenberg provides a list of reusable block supports.
+
+Let's augment our gutenberg pride block with some of these supports. To so so, we just update the `registerBlockType` call with an additional `supports` key like so:
```

Comment:
```suggestion
A lot of blocks, including core blocks, offer similar customization options. Whether that is to change the background color, text color, or to add padding, margin customization options.

To avoid duplicating the same logic over and over in your blocks and to align the behavior of your block with core blocks, Gutenberg provides a list of reusable block supports.

Let's augment our Gutenberg pride block with some of these supports. To do so, we just update the `registerBlockType` call with an additional `supports` key like so:
```